package/kernel/linux/modules/usb.mk

```diff
@@ -519,6 +519,23 @@ endef
 $(eval $(call KernelPackage,usb-dwc3))
 
 
+define KernelPackage/usb-dwc3-octeon
+  TITLE:=DWC3 Cavium Octeon USB driver
+  DEPENDS:=@(TARGET_octeon) +kmod-usb-dwc3
+  KCONFIG:= CONFIG_USB_DWC3_OCTEON
+  FILES:= $(LINUX_DIR)/drivers/usb/dwc3/dwc3-octeon.ko
+  AUTOLOAD:=$(call AutoLoad,53,dwc3-octeon,1)
```

Comment:
Considering that driver has a compatible, AutoProbe should work fine

Reply:
Just try:
`AUTOLOAD:=$(call AutoProbe,dwc3-octeon,1)`

If USB is required in pre-init then `1` before `)` must be kept.

Reply:
done